jina/docker/hubio.py

```diff
@@ -73,7 +74,46 @@ def new(self):
             self.logger.info('nothing is created, bye!')
 
     def push(self, name: str = None, readme_path: str = None):
-        """A wrapper of docker push """
+        """ A wrapper of docker push """
+        is_build_success, is_push_success = True, False
+        try:
+            # self._push(name, readme_path)
+            is_push_success = True
+        except Exception:
+            self.logger.error(f'can not push to the registry')
+        
+        file_path = f'/tmp/{self.args.name.replace("/", "_")}.json'
```

Comment:
Could we add a documentation comment for better readability that this temporary file path refers to the `build log` location?

In case this path doesn't exist, do we have a build re-try and push re-try attempt logic?

Reply:
For the cases, where this file doesn't exist, that would mean, manual deletion of the file. We don't want to rebuild during a `push` flow. Hence, fallback logic is to write whatever information we can have during a `push` only flow. I'll add a comment which says reading from tmp build log file.
Please suggest, if you feel we can have a better file to write to, rather than `/tmp/{name}.json`